package.json

```diff
@@ -48,17 +48,14 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@mdx-js/react": "^3.0.0",
     "antd": "^5.6.2",
-    "axios-retry": "^3.8.0",
     "babel-plugin-macros": "^3.1.0",
     "clsx": "^1.2.1",
     "docusaurus-plugin-image-zoom": "^2.0.0",
-    "docusaurus-plugin-openapi-docs": "^3.0.0-beta.10",
```

Comment:
This was a double entry. The old version is not needed as we are using 4.0.1 on line `55`